index.html

```diff
@@ -11,76 +11,153 @@
-        </ul>
-        <div class="buttoncontainer">
-          <a href="#professional-art" class="button">See Project</a>
-        </div>
-      </section>
-      <section id="professional-art2" class="work">
-        <h2 class="myh2tri">Professional Art Printing Data</h2>
-        <p class="Inter-text textart">A daily selection of privately personalized reads;
-        no accounts or sign-ups required. has been the industry's standard
-        </p>
-        <ul class="boxed-words2">
-          <li class="boxed-word2">html</li>
-          <li class="boxed-word2">bootstrap</li>
-          <li class="boxed-word2">Ruby</li>
-        </ul>
-        <div class="buttoncontainer">
-          <a href="#professional-art2" class="button">See Project</a>
+      <section id="multi-post-stories" class="work1">
+        <img id="gymfit" src="img/ImgPlaceholder.png" alt="Gymfit">
+        <div id="textwrapper">
+          <h2 class="myh2bis">Multi-Post Stories</h2>
+          <p class="Inter-text firstInter-text">A daily selection of privately personalized reads;
+          no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s,
+          when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="boxed-words">
+            <li class="boxed-word">css</li>
+            <li class="boxed-word">html</li>
+            <li class="boxed-word">bootstrap</li>
+            <li class="boxed-word">Ruby</li>
+          </ul>
+          <a href="#multi-post-stories" class="button">See Project</a>
         </div>
       </section>
+      <div id="workswrapper">
+        <section id="professional-art" class="pa">
+          <h2 class="myh2tri">Professional Art Printing Data</h2>
+          <p class="Inter-text textart">A daily selection of privately personalized reads;
+          no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="boxed-words2">
+            <li class="boxed-word2">html</li>
+            <li class="boxed-word2">bootstrap</li>
+            <li class="boxed-word2">Ruby</li>
+          </ul>
+          <div class="buttoncontainer">
+            <a href="#professional-art" class="button">See Project</a>
+          </div>
+        </section>
+        <section id="professional-art2" class="pa">
+          <h2 class="myh2tri">Professional Art Printing Data</h2>
+          <p class="Inter-text textart">A daily selection of privately personalized reads;
+          no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="boxed-words2">
+            <li class="boxed-word2">html</li>
+            <li class="boxed-word2">bootstrap</li>
+            <li class="boxed-word2">Ruby</li>
+          </ul>
+          <div class="buttoncontainer">
+            <a href="#professional-art2" class="button">See Project</a>
+          </div>
+        </section>
```

Comment:
- [ ] Please add the correct margin values for all work cards in the desktop version and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160032026-bd1e63d5-8897-4e6f-99f8-90d9ab4667ab.png)  | ![image](https://user-images.githubusercontent.com/68796232/160032055-30f2cc01-7205-4480-bc27-fd2908870903.png)  |

Reply:
Here I don't see the problem, you are seeing the figma design and my design in different screen widths. If we have a space that is not enough to fit a fourth card, then a blank space must lay in the right, unless I can do a space-between and even the spacings. Still, I found my inter-card spacings were bigger than they should, I corrected this. Space-around looks prettier, I enabled it.